### PR TITLE
Make mission_id 78 solvable

### PR DESCRIPTION
**Pull request by mlangsdorf**
Change mission_id 78 from "Find a Book" to "Find the Zombie Guide",
add quest dialog, and make it solvable with a copy of the Zombie
Survival Guide.

### CODE REVIEW

**Comment by mlangsdorf** [comment]
This address the bug reported here
https://github.com/CleverRaven/Cataclysm-DDA/pull/17077

**Comment by i2amroy** [comment]
The best way to handle issue fixing is to edit the original post to put in the exact phrase:
"fixes #issue_number_here"

Which will ensure that if your pull request gets merged that it automatically closes that issue.

**Comment by mlangsdorf** [comment]
On Tue, Jun 7, 2016 at 2:50 PM, Christian Buskirk <notifications@github.com>
wrote:

> The best way to handle issue fixing is to edit the original post to put in
> the exact phrase:
> "fixes #issue_number_here"
>

Does the "original post" mean the thread here:
http://smf.cataclysmdda.com/index.php?topic=11198.0

or my pull request here:
https://github.com/CleverRaven/Cataclysm-DDA/pull/17077

or something else?

And how do I get the issue number?

> Which will ensure that if your pull request gets merged that it
> automatically closes that issue.
>

I want to contribute as usefully as possible, I'm just not at all familiar
with your
workflow.

Thanks for the pointers.

--Mark Lagnsdorf

**Comment by Zireael07** [comment]
#17077 is this very PR we're discussing.
You were told to put the issue number in the PR title.

**Comment by mlangsdorf** [comment]
I will edit the PR title to add the bug issue number but I don't know the bug issue number. How do I get that information?

**Comment by Coolthulhu** [comment]
I can't test it, NPCs with this mission don't seem to want to give it. Mission code is a major mess so there's a chance missions are sorted by ID, need extra changes or something like that.

**Comment by mlangsdorf** [comment]
I branched my tree off of commit f9fa31f and at that point it was possible to test the mission using the debug key. I'm not sure what happened since then.

**Comment by BevapDin** [comment]
Well, the NPC gives out the mission once they follow you. And the mission itself works fine.

File: src/mission.cpp [hunk 1]
```diff
@@ -2496,6 +2496,32 @@ Before we get into a major fight just make sure we have the gear we need, boss."
         }
         break;
 
+    case MISSION_FIND_ZSG:
+        if( state == "TALK_MISSION_DESCRIBE" ) {
+            return _("I've can an idea about how to improve our chances of surviving");
```
Comment (BevapDin) [inline src/mission.cpp]: "can an idea"? Maybe "I've an idea..." - the "can" seems wrong there. There seems to be a period missing at the end of the sentence.

Comment (mlangsdorf) [inline src/mission.cpp]: Thanks for the feedback.

I've made the changes in my local tree to fix those two typos - how do I go about publishing them? Do I amend the commit and push the fix back to github or something else? I checked 
http://smf.cataclysmdda.com/index.php?topic=1063.0 but it didn't address the issue of making changes to your code after submitting the pull request.

Comment (Zireael07) [inline src/mission.cpp]: You push to your branch (mlangsdorf:fix_mission_78) and this PR automatically gets updated.
Pro tip: use a GUI client for git stuff, such as SourceTree or GitHub for Windows.

**Pull request closed without merging**